src/cljs/nr/gameboard/board.cljs

```diff
@@ -52,6 +52,10 @@
 
 (defonce button-channel (chan))
 
+(def any-prompt-open?
+  (let [side (:side @game-state)]
+    (get-in @game-state [side :prompt :prompt-state])))
+
```

Comment:
This will cause re-renders when game-state changes, so you should figure out how to pass a cursor in for the given side.

Reply:
not going to lie, I don't understand any part of that sentence that comes after the comma :joy: